src/vis/vis.js

```diff
@@ -33,13 +40,31 @@ var VisModel = Backbone.Model.extend({
     this.overlaysCollection = new Backbone.Collection();
   },
 
+  setOk: function () {
+    this.trigger('ok');
```

Comment:
> It's not clear to me what ok really means, it feels a bit too generic/unclear/ambiguous. What do you think about clarifying it, depending on the model?

Agree that `ok` is a bit ambiguous... What I'm trying to say with `ok` and `error` is that the "resource" was or wasn't "created/updated" successfully on the Maps API. They would be the equivalent to Backbone's `sync` and `error`. I can rename `ok` to `sync`... Is that clearer?

> Also, why not remove the duplication of state data (event+state attr), in favor of just having one data point? This also have some benefits:

The problem I see with that is that we'd need to things like:

```
vis.on('change:state', function (model, state) {
  if (state === 'error') {
    // do something
  } else if (state === 'ok') {
    // do something else
  }
});
```

I feel like the following is simpler (that's why I went with custom events):

```
vis.on('sync', function () { ... });
vis.on('error', function () { ... });
```